src/components/select/select.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+interface Props extends React.DetailedHTMLProps<React.SelectHTMLAttributes<HTMLSelectElement>, HTMLSelectElement> {
+  options: Array<string>
+}
+
+const Select: React.FC<Props> = (props: Props) => {
+  const { options } = props
+
+  return (
+    <div className="inline-block relative w-64">
+      <select
+        className="block appearance-none w-full bg-zinc-900 text-white border-2 border-gray-400 hover:border-gray-500 px-4 py-2 pr-8 rounded shadow leading-tight focus:outline-none focus:shadow-outline focus:border-purple-600"
+        role={props.name}
+        {...props}
+      >
+        {options.map((option, i) => (
```

Comment:
O que é `i`? Dê nomes que qualquer um só de ler entenda, como por exemplo `key` ao invés de `i`
```suggestion
        {options.map((option, key) => (
```

Reply:
commit: b1b7ed149602f930ee62c0d3f2e3349fd1fe848d